keps/sig-auth/0014-dynamic-audit-configuration.md

```diff
@@ -192,11 +236,12 @@ called `withDynamicAudit`. Another conditional clause will be added where the ha
 [provisioned](https://github.com/kubernetes/apiserver/blob/master/pkg/server/config.go#L536) allowing for the proper feature gating.
 
 #### Policy Enforcement
-This addition will move policy enforcement from the main handler to the backends. From the `withDynamicAudit` handler, 
-the full event will be generated and then passed to the backends. Each backend will copy the event and then be required to 
-drop any pieces that do not conform to its policy. A new sink interface will be required for these changes called `EnforcedSink`, 
-this will largely follow suite with the existing sink but take a fully formed event and the authorizer attributes as its 
-parameters. It will then utilize the `LevelAndStages` method in the policy 
+This addition will move policy enforcement from the main handler to the backends. The policy object is a scoped down 
```

Comment:
nit: Rather than "... is a scoped down version ...", I would say "... is a minimal top-level policy ..."